ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts

```diff
@@ -192,26 +176,30 @@ export class DashboardRedesignedComponent extends DashboardComponent implements
         return combineLatest([
             operatorParticipant$,
             this.governedParticipantsAgent.getGovernedStudyParticipants(this.studyGuid),
-            this.removedParticipants$
-        ]).pipe(map(([operatorParticipant, participants, removedParticipants]) => {
-            const result: DashboardParticipant[] = participants
-                .filter(({userGuid}) => !removedParticipants.includes(userGuid))
-                .map((participant, i) => ({
-                    userGuid: participant.userGuid,
-                    label: (participant.userProfile.firstName || participant.userProfile.lastName)
-                        ? `${participant.userProfile.firstName} ${participant.userProfile.lastName}`
-                        : `${this.translate.instant('Toolkit.Dashboard.ChildLabel')}${i > 0 ? ' #' + (i + 1) : ''}`,
-                }));
-            if (operatorParticipant) {
-                result.unshift({
-                    userGuid: this.session.session.userGuid,
-                    label: (operatorParticipant.profile.firstName || operatorParticipant.profile.lastName)
-                        ? `${operatorParticipant.profile.firstName} ${operatorParticipant.profile.lastName}`
-                        : this.translate.instant('Toolkit.Dashboard.UserLabel'),
-                });
-            }
-            return result;
-        }));
+        ]).pipe(
+            map(([operatorParticipant, participants]) => {
+                const result = participants.map((participant, i) => this.getUserActivitiesForParticipant(participant.userGuid)
+                    .pipe(take(1), map((activities) => ({
+                        userGuid: participant.userGuid,
+                        label: (participant.userProfile.firstName || participant.userProfile.lastName)
+                            ? `${participant.userProfile.firstName} ${participant.userProfile.lastName}`
+                            : `${this.translate.instant('Toolkit.Dashboard.ChildLabel')}${i > 0 ? ' #' + (i + 1) : ''}`,
+                        activities,
```

Comment:
I don't think you need that comma
```suggestion
                        activities
```

Reply:
Do we have styleguide about that? Just there is nothing wrong in adding comma at the last object line, it helps to not add the comma to the prev line each time we add new object property line. I beleve google style guide recommends to add comma at the end of each line in object

Reply:
OK. Makes sense that you be allowed to have comma. Thought it was enforced by compiler, but obviously not.
(We should look at the Google style guide. We DO have a style guide that another developer and I kind of put together, but that was a few years ago/old version of TypeScript and was based on TypeScript compiler itself so not typical usage.)
We should come back to the style guide thing again though....